### PR TITLE
Update conn-faq.md

### PR DESCRIPTION
**Pull request by NickStobbart**

### CODE REVIEW

File: articles/connectivity/conn-faq.md [hunk 1]
```diff
@@ -27,30 +27,28 @@ As our Assured OFFICIAL cloud platform is PSN accredited, you can also connect t
...
 
 ## How is resiliency managed on external networks?
 
-Networks such as the internet and HSCN are brought into UKCloud via diverse routes to ensure resilience of the connections. In the case of internet connectivity, we also use diverse supplier networks to ensure supplier resilience. Once within the UKCloud environment, we use Border Gateway Protocols (BGP) and our private inter-DC fibre networks to triangulate the network connections. This means that in the event of an issue with one of the networks within a single DC, we can reroute external connectivity via the second DC.
+Each community networks such as the internet, PSN, Janet and HSCN is brought into UKCloud via diverse routes to ensure resilience of the connections. In the case of internet connectivity, we also use diverse supplier networks to ensure supplier resilience. Once within the UKCloud environment, we use Border Gateway Protocols (BGP) and our private inter-DC fibre networks to triangulate the network connections. This means that in the event of an issue with one of the networks within a single DC, we can reroute external connectivity via the second DC.
```
Comment (SteveHallUKCLoud) [inline articles/connectivity/conn-faq.md:42]: internet is not a community network. Also shouldn't "networks" be singular?

"is" should be "are"

File: articles/connectivity/conn-faq.md [hunk 2]
```diff
@@ -27,30 +27,28 @@ As our Assured OFFICIAL cloud platform is PSN accredited, you can also connect t
...
+Our Assured OFFICAL and Elevated OFFICAL security domains both support SSL VPN (configured from within the edge gateway), but we do not recommend it other than for occasional remote administration purposes by a small number of administrators. If you need more extensive use of SSL VPN, we recommend you deploy a dedicated VPN appliance, such as OpenVPN.
 
 ## Can I use UKCloud as a transit or gateway between different connections?
 
-No. Our cloud platforms are designed so that all traffic either originates or terminates within our platform. We can't facilitate routing between connections. If your solution needs to service users on different networks, this will need to be implemented, managed and accredited within your application or an authorised gateway service.
+No. Our cloud platforms are designed so that all traffic either originates or terminates within our platform. We can not facilitate routing between connections. If your solution needs to service users on different networks, this will need to be implemented, managed and accredited within your application or an authorised gateway service.
```
Comment (SteveHallUKCLoud) [inline articles/connectivity/conn-faq.md:34]: are we making these changes?

File: articles/connectivity/conn-faq.md [hunk 3]
```diff
@@ -27,30 +27,28 @@ As our Assured OFFICIAL cloud platform is PSN accredited, you can also connect t
 
 ## Can I connect to my solution using SSL VPN?
 
-Our Assured OFFICAL and Elevated OFFICAL security domains both support SSL VPN (configured from within the edge gateway), but we don't recommend it other than for occasional remote administration purposes by a small number of administrators. If you need more extensive use of SSL VPN, we recommend you deploy a dedicated VPN appliance, such as OpenVPN.
+Our Assured OFFICAL and Elevated OFFICAL security domains both support SSL VPN (configured from within the edge gateway), but we do not recommend it other than for occasional remote administration purposes by a small number of administrators. If you need more extensive use of SSL VPN, we recommend you deploy a dedicated VPN appliance, such as OpenVPN.
```
Comment (SteveHallUKCLoud) [inline articles/connectivity/conn-faq.md:30]: are we making these changes?

File: articles/connectivity/conn-faq.md [hunk 4]
```diff
@@ -95,13 +93,13 @@ You can also configure your virtual firewall to allow direct remote management o
 
 ## When will UKCloud offer IPv6 addresses?
 
-We continue to work with our service providers and technology partners to ensure our platform is ready for IPv6 addressing.
+The UKCloud core network supports IPv6 addressess at the internet layer using our our own RIPE IPv6 range which we advertise to all of our internet services providers. IPv6 deployments must be discussed with UKCloud. IPv6 is not yet available for any community network (such as PSN, Janet and HSCN)
```
Comment (SteveHallUKCLoud) [inline articles/connectivity/conn-faq.md:96]: double "our"

File: articles/connectivity/conn-faq.md [hunk 5]
```diff
@@ -141,19 +139,19 @@ Yes, UKCloud does support an RLI connection for customers who have been already
 
 ## If I have an MPLS circuit direct into UKCloud, do I still have to go through a PSN connection to access the Elevated Portal?
 
-No, you don't need a PSN connection as you can access the Portal from within a VM on the Elevated platform.
```
Comment (SteveHallUKCLoud) [inline articles/connectivity/conn-faq.md:144]: are we making these changes?

File: articles/connectivity/conn-faq.md [hunk 6]
```diff
@@ -141,19 +139,19 @@ Yes, UKCloud does support an RLI connection for customers who have been already
...
+No, you do not need a PSN connection as you can access the Portal from within a VM on the Elevated platform.
 
 ## Are external Domain Name System (DNS) services available?
 
-No, we don't currently offer this service. Customers can implement their own DNS servers within their solution, or configure their virtual firewall to enable connectivity to an externally hosted DNS server (for example, those hosted on Government Secure Networks such as PSN, or those available on the internet such as Google 8.8.8.8).
+No, we do not currently offer this service. Customers can implement their own DNS servers within their solution, or configure their virtual firewall to enable connectivity to an externally hosted DNS server (for example, those hosted on Government Secure Networks such as PSN, or those available on the internet such as Google 8.8.8.8).
```
Comment (SteveHallUKCLoud) [inline articles/connectivity/conn-faq.md:146]: are we making these changes?

File: articles/connectivity/conn-faq.md [hunk 7]
```diff
@@ -141,19 +139,19 @@ Yes, UKCloud does support an RLI connection for customers who have been already
...
 
 ## Does UKCloud support Janet connectivity?
 
-Yes, UKCloud provides a Janet connection provided free for our customers. In order to utilise this network, you will need to be authorised by Jisc. There is no inbound or outbound usage charge for using the Janet connection to access the UKCloud platform.
+Yes, UKCloud provides Janet connectivity free for our customers. In order to utilise this network, you will need to be authorised by Jisc. There is no inbound or outbound usage charge for using the Janet connection to access the UKCloud platform.
```
Comment (SteveHallUKCLoud) [inline articles/connectivity/conn-faq.md:150]: Thought we were moving from Jisc?

**Review by SteveHallUKCLoud — CHANGES_REQUESTED** [review]

File: articles/connectivity/conn-faq.md (hunk 3 above)
Comment (suehighmoor) [reply articles/connectivity/conn-faq.md:30]: Happy to use non-contracted form here for emphasis

File: articles/connectivity/conn-faq.md (hunk 2 above)
Comment (suehighmoor) [reply articles/connectivity/conn-faq.md:34]: changed back to contracted form

File: articles/connectivity/conn-faq.md (hunk 1 above)
Comment (suehighmoor) [reply articles/connectivity/conn-faq.md:42]: Changed to "Networks such as the internet and PSN, Janet and HSCN community networks are"

File: articles/connectivity/conn-faq.md (hunk 4 above)
Comment (suehighmoor) [reply articles/connectivity/conn-faq.md:96]: fixed

File: articles/connectivity/conn-faq.md (hunk 5 above)
Comment (suehighmoor) [reply articles/connectivity/conn-faq.md:144]: Reverted back to contracted form

File: articles/connectivity/conn-faq.md (hunk 6 above)
Comment (suehighmoor) [reply articles/connectivity/conn-faq.md:146]: Reverted back to contracted form

File: articles/connectivity/conn-faq.md (hunk 7 above)
Comment (suehighmoor) [reply articles/connectivity/conn-faq.md:150]: Leaving in for now until I can confirm with SME. We do still provide instructions that customer needs to get authorisation from Jisc.

**Review by SteveHallUKCLoud — APPROVED** [review]
LGTM

**Pull request merged**